sources/whippet/Cargo.toml

```
@@ -0,0 +1,20 @@
+[package]
+name = "whippet"
+version = "0.1.0"
+authors = ["Matthew Yeazel <yeazelm@amazon.com>"]
+license = "Apache-2.0 OR MIT"
+edition = "2021"
+publish = false
+# Don't rebuild crate just because of changes to README.
+exclude = ["README.md"]
+
+[dependencies]
+snafu = "0.7"
+zbus = "3.8.0"
```

I'd rather not mix async runtimes:
```suggestion
zbus = { version = "3.8.0", default-features = false, features = ["tokio"] }
```